rpy/robjects/numpy2ri.py

```diff
@@ -53,14 +53,15 @@ def numpy_O_py2rpy(o):
 def numpy2rpy(o):
     """ Augmented conversion function, converting numpy arrays into
     rpy2.rinterface-level R structures. """
-    if not o.dtype.isnative:
+    if (hasattr(o.dtype, 'isnative') and not o.dtype.isnative) or \
```

Comment:
Can we have a `numpy` `o.dtype` without an attribute `isnative` ?

Reply:
yes, this is the case of the nullable integers. It might be added later though.

Reply:
<sigh>

pandas...